Housekeeping (#271)
oracle/vagrant-projects: OLCNE/README.md

> # Vagrant project to setup Oracle Linux Cloud Native Environment on Oracle Linux 7

Reviewer: ```suggestion
# Vagrant project to set up Oracle Linux Cloud Native Environment on Oracle Linux 7
```

released for Intel CPUs.

## Prerequisites

> 1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM

Reviewer: ```suggestion
1. Read the [prerequisites in the top level README](../README.md#prerequisites) to set up Vagrant with either VirtualBox or KVM
```

The VMs communicate via a private network:

- Controller node (if any): 192.168.99.100
- Master node i: 192.168.99.(100+i) / master*i*.vagrant.vm
- Worker node i: 192.168.99.(110+i) / worker*i*.vagrant.vm

## Configuration

The Vagrantfile can be used _as-is_; there are a couple of parameters you
can set to tailor the installation to your needs.

### How to configure

There are several ways to set parameters:

1. Update the Vagrantfile. This is straightforward; the downside is that you

> will loose changes when you update this repository.

Reviewer: ```suggestion
will lose changes when you update this repository.
```
